README.md

```diff
@@ -15,6 +15,13 @@ You may also be interested in the hosted TensorBoard solution at
 share your ML experiments for free. For example, [this experiment][] shows a
 working example featuring the scalar dashboard.
 
+TensorBoard can run entirely offline on your local machine, in a corporate
```

Comment:
I suggest rephrasing to "TensorBoard is designed to run entirely offline, without requiring any public access to the Internet. This may be on your local machine ..."